app/index.html

```diff
@@ -33,11 +33,13 @@
       <div class="Nav-links">
         <a href="#mission" class="Nav-link">Mission</a>
         <a href="#services" class="Nav-link">What we offer</a>
+        <a href="#newsletter" class="Nav-link">get involved</a>
         <a href="#officers" class="Nav-link">officers</a>
         <a href="#board" class="Nav-link">board</a>
-        <!-- <a href="#" class="Nav-link">experts</a> -->
+        <a href="#sponsors" class="Nav-link">sponsors and partners</a>
```

Comment:
As we discussed with @fcbaila, this link should be removed, as well as the `Mission` link.